samples/durable-functions/python/pdf-summarizer/host.json

```
@@ -16,6 +16,10 @@
       },
       "hubName": "%TASKHUB_NAME%"
     }
-  }
+  },
+  "extensionBundle": {
+    "id": "Microsoft.Azure.Functions.ExtensionBundle.Preview",
+    "version": "[4.29.0]"
```

Do we need the explicit version number or can we use the glob pattern?
```suggestion
    "version": "[4.29.0, 5.0.0)"
```